experiments/openebs/openebs-pool-network-delay/cstor_pool_network_delay.yml

```diff
@@ -0,0 +1,22 @@
+- name: Fetch the pool pod name from cvr
+  include_tasks: "/utils/apps/openebs/fetch_podname_from_cvr.yaml"
+
+- name: Get the the pool pod corresponding to the CVR
+  set_fact: 
+    cstor_pool_pod: "{{ pool_pod_named_list | random }}"
+
+- name: Display pool pod name
+  debug: 
+    msg: 
+      - "The application info is as follows:"
+      - "pod-name  : {{ cstor_pool_pod }}"
+
+# including pumba lib  -> network_chaos
+- name: Inject egress network loss on cstor-pool container for {{ chaos_duration }}ms
+  include_tasks: /chaoslib/pumba/network_chaos/network_chaos.yml
+  vars:
+    n_interface: "eth0"
```

Comment:
can we place this in openebs_components ?

Reply:
We can have it in another release. Because some experiments doesn't  have this thing.